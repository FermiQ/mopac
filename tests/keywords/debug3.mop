1SCF UHF MS=1.5 OPEN(3,3)
comment1
comment2
N 0.0 0.0 0.0

1SCF UHF MS=0.5 OPEN(3,4)
comment1
comment2
B 0.0 0.0 0.0
