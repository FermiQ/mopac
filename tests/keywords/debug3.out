 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.178M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.178M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Jul 26 17:23:37 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 3 ELECTRONS IN 3 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN = 1.5
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
1SCF UHF MS=1.5 OPEN(3,3)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         N          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    4
                           NO. OF BETA  ELECTRONS =    1

 -------------------------------------------------------------------------------
 1SCF UHF MS=1.5 OPEN(3,3)
 comment1
 comment2


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.178M)
                                                       Mon Jul 26 17:23:37 2021




          FINAL HEAT OF FORMATION =       1734.48942 KCAL/MOL =    7257.10372 KJ/MOL


          TOTAL ENERGY            =       -111.36552 EV
          ELECTRONIC ENERGY       =       -111.36552 EV  POINT GROUP:     R3  
          CORE-CORE REPULSION     =          0.00000 EV
          COSMO AREA              =         42.08 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.67 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         49.742258 EV
          ALPHA SOMO LUMO (EV)    =        -49.742-32.783
          BETA  SOMO LUMO (EV)    =        -49.742-32.783
          NO. OF ALPHA ELECTRONS  =          4
          NO. OF BETA  ELECTRONS  =          1
          MOLECULAR WEIGHT        =         14.0067


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.027 SECONDS
          COMPUTATION TIME        =          0.027 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       N          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1    N        0.000000000     0.000000000     0.000000000


           Empirical Formula: N  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
 -49.74226 -32.78301 -32.78301 -32.78301


           BETA EIGENVALUES 
 -49.74226 -32.78301 -32.78301 -32.78301


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          N           3.000000        2.0000     2.00000     0.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  N   2.00000   0.00000   0.00000   0.00000


                    (SZ)    =    1.500000
                    (S**2)  =    2.250000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  N  0.0000  0.000  0.000  0.000  0.000
 *******************************************************************************
 ** Site#:    0         For non-commercial use only           Version 21.178M **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.178M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web-site: HTTP://OpenMOPAC.net.          **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Jul 26 17:23:37 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 3 ELECTRONS IN 4 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN = 0.5
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
1SCF UHF MS=0.5 OPEN(3,4)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       B          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         B          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: B  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    2
                           NO. OF BETA  ELECTRONS =    1

 -------------------------------------------------------------------------------
 1SCF UHF MS=0.5 OPEN(3,4)
 comment1
 comment2


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.178M)
                                                       Mon Jul 26 17:23:37 2021




          FINAL HEAT OF FORMATION =        708.07660 KCAL/MOL =    2962.59249 KJ/MOL


          TOTAL ENERGY            =        -26.01741 EV
          ELECTRONIC ENERGY       =        -26.01741 EV  POINT GROUP:     R3  
          CORE-CORE REPULSION     =          0.00000 EV
          COSMO AREA              =         52.81 SQUARE ANGSTROMS
          COSMO VOLUME            =         36.09 CUBIC ANGSTROMS

          BETA  SOMO LUMO (EV)    =        -26.152-14.682
          NO. OF ALPHA ELECTRONS  =          2
          NO. OF BETA  ELECTRONS  =          1
          MOLECULAR WEIGHT        =         10.8100


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.055 SECONDS
          COMPUTATION TIME        =          0.050 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       B          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1    B        0.000000000     0.000000000     0.000000000


           Empirical Formula: B  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
 -20.00669 -13.50284 -13.45214 -13.45214


           BETA EIGENVALUES 
 -26.15157 -14.68237 -14.32256 -14.32256


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          B           2.000000        1.0000     0.95000     0.05000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  B   0.95000   0.05000   0.00000   0.00000


                    (SZ)    =    0.500000
           Average over configurations used, so (S**2) is not meaningful


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  B -1.0000 -0.950 -0.050  0.000  0.000

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.10 SECONDS

 == MOPAC DONE ==
